test/mlir/test/gc/Transforms/deepTileContractionNamedOp.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: gc-opt --split-input-file --deep-tile-contraction-named-op %s | FileCheck %s
+// RUN: gc-opt --split-input-file --deep-tile-contraction-op %s | FileCheck %s
```

Comment:
Also add a generic version in the test cases?

Reply:
Added